core/src/test/kotlin/org/evomaster/core/output/TestCaseWriterTest.kt

```diff
@@ -0,0 +1,289 @@
+    }
+
+
+    private fun buildEvaluatedIndividual(dbInitialization: MutableList<DbAction>): Triple<OutputFormat, String, EvaluatedIndividual<RestIndividual>> {
+        val format = OutputFormat.JAVA_JUNIT_4
+
+        val baseUrlOfSut = "baseUrlOfSut"
+
+        val sampleType = SampleType.RANDOM
+
+        val restActions = ArrayList<RestAction>()
+
+
+        val individual = RestIndividual(restActions, sampleType, dbInitialization)
+
+        val fitnessVal = FitnessValue(0.0)
+
+        val results = ArrayList<ActionResult>()
+
+        val ei = EvaluatedIndividual<RestIndividual>(fitnessVal, individual, results)
+        return Triple(format, baseUrlOfSut, ei)
+    }
+
+    @Test
+    fun testTwoAction() {
+        val aColumn = Column("aColumn", "VARCHAR", 10)
+
+        val aTable = Table("myTable", setOf(aColumn), HashSet<ForeignKey>())
+
+
+        val id = 0L
+
+        val gene0 = StringGene(aColumn.name, "stringValue0", 0, 10)
+
+        val insertIntoTableAction0 = DbAction(aTable, setOf(aColumn), id, mutableListOf(gene0))
+
+        val gene1 = StringGene(aColumn.name, "stringValue1", 0, 10)
+
+        val insertIntoTableAction1 = DbAction(aTable, setOf(aColumn), id, mutableListOf(gene1))
+
+
+        val (format, baseUrlOfSut, ei) = buildEvaluatedIndividual(mutableListOf(insertIntoTableAction0, insertIntoTableAction1))
+
+        val test = TestCase(test = ei, name = "test")
+
+        val writer = TestCaseWriter()
+
+        val lines = writer.convertToCompilableTestCode(format, test, baseUrlOfSut)
+
+        val expectedLines = Lines()
+        expectedLines.add("@Test")
+        expectedLines.add("public void test() throws Exception {")
+        expectedLines.indent()
+        expectedLines.add("List<InsertionDto> insertions = sql().insertInto(\"myTable\").d(\"aColumn\", \"stringValue0\").and().insertInto(\"myTable\").d(\"aColumn\", \"stringValue1\").dtos();")
+        expectedLines.add("controller.execInsertionsIntoDatabase(insertions);")
+        expectedLines.deindent()
+        expectedLines.add("}")
+
+        assertEquals(expectedLines.toString(), lines.toString())
+    }
+
+
+    @Test
+    fun testTwoColumns() {
+        val column0 = Column("Column0", "VARCHAR", 10)
+        val column1 = Column("Column1", "VARCHAR", 10)
+
+        val aTable = Table("myTable", setOf(column0, column1), HashSet<ForeignKey>())
+
+        val id = 0L
+
+        val gene0 = StringGene(column0.name, "stringValue0", 0, 10)
+        val gene1 = StringGene(column1.name, "stringValue1", 0, 10)
+
+        val insertIntoTableAction = DbAction(aTable, setOf(column0, column1), id, mutableListOf(gene0, gene1))
+
+        val (format, baseUrlOfSut, ei) = buildEvaluatedIndividual(mutableListOf(insertIntoTableAction))
+
+        val test = TestCase(test = ei, name = "test")
+
+        val writer = TestCaseWriter()
+
+        val lines = writer.convertToCompilableTestCode(format, test, baseUrlOfSut)
+
+        val expectedLines = Lines()
```

Comment:
this can be simplified with ".apply{}", so you do not have to repeat expectedLines